From: Werner Koch <wk@gnupg.org>
Date: Tue, 28 Feb 2017 09:35:41 +0100
Subject: gpgv,w32: Fix --status-fd.

* g10/gpgv.c (main): Use translate_sys2libc_fd_int for --status-fd.

Signed-off-by: Werner Koch <wk@gnupg.org>
(cherry picked from commit 8a67dc4c4324b617b5a3fea51c59c674488544d6)
---
 g10/gpgv.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/g10/gpgv.c b/g10/gpgv.c
index bd16b3907..ca8fca423 100644
--- a/g10/gpgv.c
+++ b/g10/gpgv.c
@@ -194,7 +194,9 @@ main( int argc, char **argv )
           break;
         case oKeyring: append_to_strlist( &nrings, pargs.r.ret_str); break;
         case oOutput: opt.outfile = pargs.r.ret_str; break;
-        case oStatusFD: set_status_fd( pargs.r.ret_int ); break;
+        case oStatusFD:
+          set_status_fd (translate_sys2libc_fd_int (pargs.r.ret_int, 1));
+          break;
         case oLoggerFD:
           log_set_fd (translate_sys2libc_fd_int (pargs.r.ret_int, 1));
           break;
